internal/app/relayer/domain/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"github.com/bianjieai/tibc-relayer-go/internal/app/relayer/repostitory"
+	"github.com/jasonlvhit/gocron"
```

Comment:
fmt, guide: 
There should be two import groups:

- Standard library
- Everything else